src/actions/google/drive/google_drive.ts

```diff
@@ -368,6 +400,22 @@ export class GoogleDriveAction extends Hub.OAuthAction {
     return google.drive({version: "v3", auth: client})
   }
 
+  protected async checkDomain(redirect: string, tokens: Credentials, domainList: string) {
+    const list = domainList.split(",")
```

Comment:
Should we think about a more robust way to parse the string? One that can handle extra whitespace and other input issues?

`google.com,@googlea.com, gmail.com  ,,  , `